unreal_asset/src/exports/level_export.rs

```diff
@@ -80,21 +98,32 @@ impl ExportTrait for LevelExport {
 
         asset.write_i32::<LE>(0)?;
         asset.write_i32::<LE>(self.actors.len() as i32)?;
-        for actor in &self.actors {
+        for actor in self.actors.iter() {
```

Comment:
why the change to iter?

Reply:
idk I personally think it's nicer - it's the same thing under the hood anyway but I can change it back if you want

Reply:
most of the code uses `&` so it's nicer if you would change it back for consistency

Reply:
sure